contrib/cirrus/prebuild.sh

```diff
@@ -77,12 +77,25 @@ if [[ "${DISTRO_NV}" == "$PRIOR_FEDORA_NAME" ]]; then
     # simply here to...
     msg "Checking GitLeaks functions with current CLI args, configuration, and baseline JSON"
 
+    # TODO: Workaround for GHA Environment, duplicate here for consistency.
+    # Replace with `--userns=keep-id:uid=1000,gid=1000` w/ newer podman in GHA environment.
+    declare -a workaround_args
+    workaround_args=(\
+      --user 1000:1000
+      --uidmap 0:1:1000
+      --uidmap 1000:0:1
+      --uidmap 1001:1001:64536
+      --gidmap 0:1:1000
+      --gidmap 1000:0:1
+      --gidmap 1001:1001:64536
+    )
+
     brdepth=$(get_env_key 'brdepth')
     glfqin=$(get_env_key 'glfqin')
     glargs=$(get_env_key 'glargs')
     showrun podman run --rm \
         --security-opt=label=disable \
-        --userns=keep-id:uid=1000,gid=1000 \
+        "${workaround_args[@]}" \
```

Comment:
could we just drop it?

Reply:
We could, I'm slightly apprehensive about carrying around the added complexity forever.  Another minor issue is if the gitsecrets project changes their container to use something other than UID `1000`, it'd be a pain to fix this workaround vs just updating a single line.